### DISCUSSION TITLE
Toast to the StarSearch Launch 5/24/2024

### DISCUSSION THREAD

**Discussion by bdougie** [open-sauced/feedback#32] [Show and tell]
The last 2 months have been wild. Starting with React Miami the team got together to talk through AI Engineering ourselves an interaction that provides even more context from our opens source data. I am happy to share we are now all competent AI engineers and ready to take this to the next level. 

Read about StarSearch [here](https://opensauced.pizza/blog/open-source-insights-with-starsearch)

## How to toast
This is the fun part. 

1. Take a selfie with a pizza or drink
2. Post as a comment below with a shout to someone or the team that made this happen.

**Comment by nickytonline** [comment]
Wow! What a ride it's been. We've been shipping so much goodness and even though it's been a lot of work, I can genuinely say it's been a lot of fun. We have a great crew and I'm proud to raise a slice of pie!

Big ups to @open-sauced/engineering, @open-sauced/design and @open-sauced/staff. And big ups to @BekahHW for being our fearless Lunch Week captain!

![image](https://github.com/open-sauced/feedback/assets/833231/8db4f576-2fd3-4c51-a219-1e09f8d9cad2)

This was my first time ordering from [Chicago Pizza](http://chicagopizzamontreal.ca/). They did not disappoint. Pictured is their meat pizza which includes pepperoni, sausage, bacon, and ham.

**Comment by bdougie** [comment]
Shout out to @open-sauced/staff and @open-sauced/community for an amazing week. It is truly an honor to work with so many cool people and impact open source in the biggest way! 

![Screenshot 2024-05-24 at 10 07 33 AM](https://github.com/open-sauced/feedback/assets/5713670/02dfe55d-d38b-416e-8c7a-d6882743a018)

"[who's the boss](https://www.arthurmacs.com/meaty-pies)" slice frmo arthur macs in Oakland.

**Comment by jpmcb** [comment]
Cheers to the whole OpenSauced team and our community!!! It's been an incredible journey building this tooling for the open-source ecosystem and it's been a joy to see us ship some of these really big, amazing features!

<img width="982" alt="Screenshot 2024-05-24 at 2 22 50 PM" src="https://github.com/open-sauced/feedback/assets/23109390/36872c0d-b729-4591-935e-0f5d75bd40f3">

- @open-sauced/engineering: great work on all the big features we shipped - from AI to visualizations, new components, new routes and capabilities in the API, and fixes all around. The app is feeling real _fast_ these days!!!! Great work!
- @open-sauced/design: where would we be without the amazing designs from Isa. Thank you for your thoughtful approach to design and product direction!
- @open-sauced/staff: the ship wouldn't stay afloat without any one of us. Great work on this release and great work on making this all possible!!!

**Comment by BekahHW** [comment]
![image](https://github.com/open-sauced/feedback/assets/34313413/76e856cc-15df-422a-8388-8c04cdb6fcdf)

[Ohio Valley Pizza](https://en.wikipedia.org/wiki/Ohio_Valley%E2%80%93style_pizza) all the way.

To the @open-sauced/staff: This has been a _wild_ ride over the last couple of weeks and you did a great job of continually allowing us to grow. 

- To @nickytonline thank you for always supporting my big ideas, even when it's an exhausting 3-hr stream (I won't make you do that again. lol). I'm thankful for your continual support. 
- To @jpmcb for answering questions, drafting thoughtful blog posts, and for being willing to explore. I'm grateful to be on this adventure with you.
- To @open-sauced/docs: You all amaze me with your thoughtfulness and willingness to make open source better for everyone who's getting started. You crushed it this week, whether you know it or not.

**Reply by bdougie** [reply]
Can't wait for the Steubenville company offsite so we can try this pizza.

**Comment by chhristopher** [comment]
Shoutout to @open-sauced/staff: Our team delivers results like pizza delivery – hot, fresh, and always on time! Proud of ya'll. Ordering pizza later but let me put you on some Shin Ramen. 

<img width="662" alt="Screenshot 2024-05-24 at 5 00 09 PM" src="https://github.com/open-sauced/feedback/assets/48575398/a0f243f5-e00d-4858-a45d-47468f88b455">

**Comment by zeucapua** [comment]
![Photo on 5-24-24 at 18 15](https://github.com/open-sauced/feedback/assets/20603494/f32331b0-9995-4513-a24e-f4b99008ee30)

[Settebello Pizzeria Napoletana](https://www.settebello.net/) is top tier for me!

This is my first launch week and my goodness it was hectic, but it was all worth it. It couldn't have been done with everyone in the @open-sauced/staff team! I'm very optimistic for the future of our product and I can't wait to work on features for our users.

**Comment by adiati98** [comment]
![ayu-with-croissant](https://github.com/open-sauced/feedback/assets/45172775/83f271e1-2bf8-4be0-9985-24d915c8ccdb)


I have no pizza, so here is a croissant! 🥐 

Big shoutout to @open-sauced/staff for the super big launch(es)! Y'all awesome!!! 🙌🏼 🍕 

Huge shoutout to @open-sauced/docs team for the great teamwork and thoughts in baking Becoming a Maintainer Course! It was one great ride and an invaluable experience. I'm grateful to be part of it! 🫶

**Reply by nickytonline** [reply]
Love the croissant Ayu! Thanks so much for all you've done with docs, the Becoming a Maintainer Course and the community! 🍕

**Comment by higgn** [comment]


**Reply by higgn** [reply]
@bdougie @nickytonline @zeucapua @jpmcb @BekahHW

**Reply by nickytonline** [reply]
Can you file an issue at https://github.com/open-sauced/app/issues/new?assignees=&labels=%F0%9F%91%80+needs+triage%2C%F0%9F%90%9B+bug&projects=&template=bug_report.yml&title=Bug%3A+ ? Thanks!

**Reply by BekahHW** [reply]
@Zesina thanks for the heads up. Can you create an issue in https://github.com/open-sauced/app?

**Reply by higgn** [reply]
> @Zesina thanks for the heads up. Can you create an issue in https://github.com/open-sauced/app?

surely.. I will do that :)

**Reply by higgn** [reply]
> Can you file an issue at https://github.com/open-sauced/app/issues/new?assignees=&labels=%F0%9F%91%80+needs+triage%2C%F0%9F%90%9B+bug&projects=&template=bug_report.yml&title=Bug%3A+ ? Thanks!

Surely sir (:
